src/ai/backend/manager/registry.py

```diff
@@ -2304,7 +2304,11 @@ async def _update() -> None:
                     else:
                         log.error('should not reach here! {0}', type(row['status']))
 
-            await execute_with_retry(_update)
+            try:
+                await execute_with_retry(_update)
+            except sa.exc.IntegrityError:
+                log.error(f'Scaling group named [{sgroup}] does not exist.')
+                await asyncio.sleep(5.0)
```

Comment:
Could you explain what this sleep is for?

Reply:
It is for delaying error message. When error raises without sleep, that makes too many messages.

Reply:
Hm... is the situation that only happens when restarting the manager after a while with keeping agents running? That's because the heartbeat events are accumulated in Redis and then fetched at the maximum speed upon manager restart. It's an expected behavior.

We need to solve that issue in a different way, possibly adding a wall clock timestamp in heartbeats and ignore "too old" heartbeats, or aggregate multiple heartbeats from same agents into one (leaving only the last one) before processing it.
I'm not sure how to achieve this; would it better to put such "burst-control" feature in the event bus or just implement in the heartbeat handler?